config/ccms/means_entity_configs/non_passported.yml

```diff
@@ -0,0 +1,142 @@
+  entity_name: BANKACC
+  yaml_section: :bank_acct
+
+- predicate: :applicant_recieves_financial_support?
+  method: generate_entity
+  entity_name: CLIENT_FINANCIAL_SUPPORT
+  instance_label: the client's financial support1
+  yaml_section: :financial_support
+
+- predicate: true
+  method: generate_entity
+  entity_name: CHANGE_IN_CIRCUMSTANCE
+  instance_label: the change in circumstance
+  yaml_section: :change_in_circumstance
+
+- predicate: :vehicles_present?
+  method: :generate_vehicles_entity
+  entity_name: CARS_AND_MOTOR_VEHICLES
+  instance_label: the car and motor vehicle
+  yaml_section: :vehicles
+
+- predicate: :applicant_has_access_to_other_person_account?
+  method: :generate_entity
+  entity_name: THIRDPARTACC
+  instance_label: the car and motor vehicle
+  yaml_section: :third_party_accounts
+
+- predicate: applicant_has_premium_bonds?
+  method: :generate_entity
+  entity_name: CLIPREMIUM
+  instance_label: the client's premium bond1
+  yaml_section: :premium_bonds
+
+- predicate: applicant_has_money_due?
+  method: :generate_entity
+  entity_name: MONEY_DUE
+  instance_label: the money due1
+  yaml_section: :money_due
+
+- method: :generate_means_proceeding_entity
+  entity_name: PROCEEDING
+  yaml_section: :proceeding_means
+
+- predicate: applicant_has_capital_bonds?
+  method: :generate_entity
+  entity_name: CLICAPITAL
+  instance_label: the money due1
+  yaml_section: :cli_capital
+
+- method: :generate_entity
+  entity_name: OPPONENT_OTHER_PARTIES
+  instance_label: OPPONENT_7713451
+  yaml_section: :opponent_other_parties_means
+
+- method: :generate_entity
+  entity_name: LAND
+  instance_label: the land
+  yaml_section: :land
+
+- method: :generate_entity
+  entity_name: TIMESHARE
+  instance_label: the timeshare
+  yaml_section: :timeshare
+
+- method: :generate_entity
+  entity_name: global
+  instance_label: "#submission_case_ccms_reference"
+  yaml_section: :global_means
+
+- method: :generate_entity
+  entity_name: OTHERSAVING
+  instance_label: the timeshare
+  yaml_section: :other_savings
+
+- predicate: :vehicles_present?
+  method: :generate_entity
+  entity_name: CAR_USED
+  instance_label: the car used
+  yaml_section: :car_used
+
+- predicate: applicant_has_second_home?
+  method: :generate_entity
+  entity_name: ADDPROPERTY
+  instance_label: "#add_property_instance_label"
+  yaml_section: :additional_property
+
+- predicate: applicant_has_shares?
+  method: :generate_entity
+  entity_name: CLISTOCK
+  instance_label: the client's stock1
+  yaml_section: :stocks
+
+- predicate: applicant_has_life_assurance?
+  method: :generate_entity
+  entity_name: LIFE_ASSURANCE
+  instance_label: the client's life assurance
+  yaml_section: :life_assurance
+
+- predicate: applicant_owns_share_of_property?
+  method: :generate_entity
+  entity_name: MAINTHIRD
+  instance_label: the third party who part owns the main dwelling1
+  yaml_section: :main_dwelling_third_party
+
+- predicate: true
```

Comment:
This predicate doesn't match the others, should it?

Reply:
Yep - it just means always generate the entity block - don't rely on the result of a method to tell you whether or not to do it.